src/lib/form-field/form-field.html

```diff
@@ -1,6 +1,21 @@
 <div class="mat-form-field-wrapper">
   <div class="mat-form-field-flex" #connectionContainer
        (click)="_control.onContainerClick && _control.onContainerClick($event)">
+
+    <!-- Outline used for  outline appearance. -->
```

Comment:
nit: there's an extra space between "for" and "outline".

Reply:
Removed extra space in new commit. Thanks @crisbeto!